src/Commands/Servers/Make.php

```diff
@@ -39,9 +42,12 @@ public function configure()
             ->addOption('private-ip', null, InputOption::VALUE_REQUIRED, 'Private IP address of the server. Required when using the "custom" provider.')
             ->addOption('php', null, InputOption::VALUE_REQUIRED, 'PHP version to install when provisioning the server. Supported versions are "php56", "php70", and "php71".')
             ->addOption('database', null, InputOption::VALUE_REQUIRED, 'The name of the database to create when provisioning the server.', 'forge')
+            ->addOption('database-type', null, InputOption::VALUE_OPTIONAL, 'The type of DB to install. Can be "mariadb", "mysql", "mysql8, "postgres", or "none".', 'mysql')
             ->addOption('install-maria', null, InputOption::VALUE_NONE, 'Whether MariaDB should be installed. If left out, MySQL will be installed by default.')
             ->addOption('load-balancer', null, InputOption::VALUE_NONE, 'Whether the server should be provisioned as a load balancer.')
             ->addOption('network', null, InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED, 'Other servers\' ids this one can network with.')
+            ->addOption('name', null, InputOption::VALUE_REQUIRED, 'The name of the server.')
+            ->addOption('size', null, InputOption::VALUE_REQUIRED, 'The size of the server.')
```

Comment:
Should we let the user know what this "size" means? It looks like the docs mention it's provider- and region specific, so we should probably mention that in the "help" text here as well:

![image](https://user-images.githubusercontent.com/11269635/86512930-39fa2f00-be06-11ea-90e6-e14b34950693.png)

https://forge.laravel.com/api-documentation#regions

Reply:
That's probably a good idea.